e2e-tests/insurance/cypress/e2e/journeys/your-buyer/root/your-buyer-root.spec.js

```diff
@@ -0,0 +1,126 @@
+import partials from '../../../../../../partials';
+import {
+  listIntro,
+  listItem,
+  listOutro,
+  outro,
+  startNowLink,
+  allSectionsLink,
+} from '../../../../../../pages/shared';
+import { PAGES, BUTTONS } from '../../../../../../content-strings';
+import { INSURANCE_ROUTES } from '../../../../../../constants/routes/insurance';
+
+const CONTENT_STRINGS = PAGES.INSURANCE.YOUR_BUYER.ROOT;
+
+const {
+  ROOT,
+  ALL_SECTIONS,
+  YOUR_BUYER: {
+    ROOT: YOUR_BUYER_ROOT,
+    COMPANY_OR_ORGANISATION,
+  },
+} = INSURANCE_ROUTES;
+
+const { taskList } = partials.insurancePartials;
+
+const task = taskList.prepareApplication.tasks.buyer;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context('Insurance - Your buyer - start page - As an exporter, I want to provide the details on the buyer of my export trade, So that UKEF can gain clarity on the buyer history as part of due diligence', () => {
```

Comment:
Capital S for start page? not a blocker

Reply:
Great spot - I have a small tech improvement for "section start" controllers after this, will address there.